xbmc/pvr/PVRGUIInfo.cpp

```diff
@@ -191,7 +191,10 @@ bool CPVRGUIInfo::TimerInfoToggle(void)
 
 void CPVRGUIInfo::Process(void)
 {
-  unsigned int mLoop(0);
+  unsigned int mLoop(1); /* Delay first backend data cache update. It might last
```

Comment:
not really caused by this PR, bu if you use hungarian notation, use it correctly ;-) `m_` is for members